src/components/date-range/date-range.pw.tsx

```diff
@@ -0,0 +1,874 @@
+    });
+  });
+
+  testData.forEach((endError) => {
+    test(`should check accessibility with the endError as string renders as ${endError}`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<DateRangeCustom endError={endError} />);
+
+      await checkAccessibility(page);
+    });
+  });
+
+  testData.forEach((startWarning) => {
+    test(`should check accessibility with the startWarning as string renders as ${startWarning}`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<DateRangeCustom startWarning={startWarning} />);
+
+      await checkAccessibility(page);
+    });
+  });
+
+  testData.forEach((endWarning) => {
+    test(`should check accessibility with the endWarning as string renders as ${endWarning}`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<DateRangeCustom endWarning={endWarning} />);
+
+      await checkAccessibility(page);
+    });
+  });
+
+  testData.forEach((startInfo) => {
+    test(`should check accessibility with the startInfo as string renders as ${startInfo}`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<DateRangeCustom startInfo={startInfo} />);
+
+      await checkAccessibility(page);
+    });
+  });
+
+  testData.forEach((endInfo) => {
+    test(`should check accessibility with the endInfo as string renders as ${endInfo}`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<DateRangeCustom endInfo={endInfo} />);
+
+      await checkAccessibility(page);
+    });
+  });
+
+  testData.forEach((error) => {
+    test(`should check accessibility with the validationOnLabel with error state as ${error}`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(
+        <DateRangeCustom
+          startError={error}
+          endError={error}
+          validationOnLabel
+        />
+      );
+
+      await checkAccessibility(page);
+    });
+  });
+
+  testData.forEach((warning) => {
+    test(`should check accessibility with the validationOnLabel with warning state as ${warning}`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(
+        <DateRangeCustom
+          startWarning={warning}
+          endWarning={warning}
+          validationOnLabel
+        />
+      );
+
+      await checkAccessibility(page);
+    });
+  });
+
+  testData.forEach((info) => {
+    test(`should check accessibility with the validationOnLabel with info state as ${info}`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(
+        <DateRangeCustom startInfo={info} endInfo={info} validationOnLabel />
+      );
+
+      await checkAccessibility(page);
+    });
+  });
+
+  ([
+    "top",
+    "bottom",
+    "left",
+    "right",
+  ] as DateRangeProps["tooltipPosition"][]).forEach((position) => {
+    test(`should check accessibility with the tooltipPosition is set to ${position}`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(
+        <DateRangeCustom
+          m={9}
```

Comment:
**question**: do we have to pass margin here?

Reply:
Yeah we have to for the tooltip positions. We need room all around the component so that the tooltip can render in the correct position.